javaagent/build.gradle.kts

```diff
@@ -0,0 +1,157 @@
+import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
+import com.github.jk1.license.filter.LicenseBundleNormalizer
+import com.github.jk1.license.render.InventoryMarkdownReportRenderer
+
+plugins {
+  id("com.github.jk1.dependency-license-report")
+
+  id("otel.java-conventions")
+  id("otel.publish-conventions")
+  id("otel.shadow-conventions")
+}
+
+description = "OpenTelemetry Javaagent"
+
+group = "io.opentelemetry.javaagent"
+
+val shadowInclude by configurations.creating {
+  isCanBeResolved = true
+  isCanBeConsumed = false
+}
+
+fun isolateSpec(projectsWithShadowJar: Collection<Project>): CopySpec = copySpec {
```

Comment:
In a future PR I am going to try to move this to a DSL extension on `otel.shadow-conventions`